software_steering_council.md

```diff
@@ -0,0 +1,52 @@
+# Software Steering Council
+
+## Purpose
+
+The Jupyter Software Steering Council (SSC) has jurisdiction over software-related decisions across Project Jupyter, with a primary focus on coordination across projects and decisions that have impact across many Jupyter Subprojects. It is also a mechanism for representatives of each project to share information and expertise. Technical decisions and processes where the SSC isn't explicitly involved are automatically delegated to the individual projects to manage their day-to-day activities, create new repositories in their orgs, etc., with independence and autonomy.
+
+## Responsibilities
+
+### Sole responsibilities
+
+The SSC is responsible for the following:
+
+- Defining the submission, review, and approval process for Jupyter Enhancement Proposals.
+- Stewarding the JEP process to ensure that it is inclusive and participatory, and solicits feedback from the right stakeholders. As a starting point, [the NumFOCUS DISCOVER Cookbook](https://github.com/numfocus/DISCOVER-Cookbook) may offer some useful starting pointers.
+- Making decisions on JEPs after receiving community feedback (JEPs are the mechanism for elevating any software decision/idea up to the broader Project, while individual Subprojects can make decisions within their scope via their regular mechanisms).
+- Owning, managing and implementing the Jupyter Incubation process (https://github.com/jupyter-incubator).
+- Owning, managing and implementing the Jupyter Attic process (https://github.com/jupyter-attic).
+- Stewarding discussions and activities beyond JEPs that involve cross-cutting concerns, standards, protocols and other architectural issues that impact the entire project.
+- Owning and managing anything related to security vulnerabilities across the project, including the Jupyter security lists, any private security repositories, etc.
+
+### Shared responsibilities with the Board of Directors
+
+The SSC and BoD share the responsibility for:
+
+- Updates or amendments to the [Jupyter Code of Conduct](https://jupyter.org/governance/conduct/code_of_conduct.html).
+- Changing the Jupyter Governance model.
+- Creation of new Jupyter Subprojects.
+- Removal of Subprojects from Jupyter.
+
+## Membership
+
+SSC members are leaders from Jupyter Subprojects that wish to assist the community in sharing information across projects and participate in decision-making that affects many stakeholders in the Jupyter ecosystem. Membership requires active and ongoing participation in the project.
+
+### Member composition
+
+The Software Steering Council is composed of one representative from each of the Jupyter Subprojects. In addition, the SSC may contain members from certain working groups that have been created by the BoD and whose scope has important impact on SSC activities, such as Diversity and Inclusion, Accessibility, and Internationalization.
```

Comment:
While this somewhat addresses my concern above, it doesn't go far enough. "may" should be changed to "must"

Reply:
Correct, that was the intent all along and in fact we updated the language to "will".